modules/nback.llvm/llvm.bal

```diff
@@ -56,15 +56,23 @@ public distinct class BasicBlock {
     private final string label;
     private final string[] lines;
 
-    function init(string label, Function func) {
+    function init(string label, Function func, boolean ignoreLabel = false) {
```

Comment:
The ignoreLabel stuff is ugly. How do you ensure that the API doesn't refer to the label?

Seems better to check whether a label is every referenced in another instruction, and if not don't include it.

Reply:
For the first basic block in the function there shouldn't be the label in the output. So it should be like
```
define i64 @main (  ) {
  ret i64 0
}
```
Not 
```
define i64 @main (  ) {
L0:
  ret i64 0
}
```

Reply:
1. How do you guarantee that the API user cannot create a reference to this label?
2. The label is not necessary in this case, but llvm-as accepts it. Removing it is just prettification.